packages/order-watcher/test/event_watcher_test.ts

```diff
@@ -1,126 +0,0 @@
-import { callbackErrorReporter } from '@0xproject/dev-utils';
```

Comment:
These tests no longer make sense not that we just use blockstream under-the-hood.